src/main/java/org/owasp/html/ExtensibleHtmlStreamRenderer.java

```diff
@@ -143,7 +143,7 @@ protected ExtensibleHtmlStreamRenderer(Appendable output, Handler<? super IOExce
      * @param message    for human consumption.
      * @param identifier an HTML identifier associated with the message.
      */
-    private final void error(String message, CharSequence identifier) {
+    public final void error(String message, CharSequence identifier) {
```

Comment:
Protected not public. And if you make a final then it can not be overridden

Reply:
done , and for other methodd as well
changed from private to protected and removed final